### DISCUSSION TITLE
Alternative to latex's \dotfill

### DISCUSSION THREAD

**Discussion by KillTheMule** [typst/typst#2023] [General]
As the title says, I'm looking to use something akin to latex's `\dotfill` in typst. It simply fills up the remaining space of a line with dots (most importantly, in tables, too, so that it fills up the remaining space of a cell with dots). I tried variations of `#line()` to no avail, and google was of no help.

Thanks for any pointers :)

**Comment by Enivex** [comment]
https://typst.app/docs/reference/layout/repeat/

This might interest you

**Reply by KillTheMule** [reply]
Seems very relevant, thanks a bunch!
